app/views/provider_interface/start_page/show.html.erb

```diff
@@ -40,7 +40,11 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-one-half">
         <h2 class="govuk-heading-m">View applications</h2>
-        <p class="govuk-body">See the status of applications at a glance.</p>
+        <p class="govuk-body">See newly submitted applications at a glance.</p>
+        <p class="govuk-body">Filter by status, training provider, ratifying body – and soon, reject by default date.</p>
+      <% if FeatureFlag.active?('provider_make_decisions_restriction') && FeatureFlag.active?('provider_view_safeguarding') %>
```

Comment:
```suggestion
      <% if FeatureFlag.active?('provider_add_provider_users') %>
```

This is the flag which actually turns on the UI for providers. It won't be turned on until the others are.